src/server/utils/requester/requester.ts

```diff
@@ -56,7 +57,8 @@ export function properRequesterFactory(options: ProperRequesterOptions): Plywood
       requester = druidRequesterFactory({
         host,
         timeout: timeout || 30000,
-        requestDecorator: options.druidRequestDecorator
+        requestDecorator: options.druidRequestDecorator,
+        protocol: options.protocol
```

Comment:
`druidRequesterFactory` accepts `DruidRequesterParameters` from `plywood-druid-register`. There, `protocol` field is declared as `export declare type Protocol = 'plain' | 'tls-loose' | 'tls'`

https://github.com/implydata/plywood-druid-requester/blob/master/src/druidRequester.ts#L29

Reply:
Is it sufficient to cast it to the Protocol type?

Reply:
Ideally I would define `protocol` field in our code with type `Protocol` from `plywood-druid-register`. They're exporting it. That way we also could define default value in our code (probably to 'plain').